import System.Exit

main :: IO ()
main = do
  -- should have some tests
  exitFailure

